linux/LinuxProcessList.c

```diff
@@ -192,13 +206,30 @@ static void LinuxProcessList_initTtyDrivers(LinuxProcessList* this) {
    this->ttyDrivers = ttyDrivers;
 }
 
+#ifdef HAVE_DELAYACCT
+
+static void LinuxProcessList_initNetlinkSocket(LinuxProcessList* this) {
+  this->netlink_socket = nl_socket_alloc();
+  if (this->netlink_socket == NULL)
+    return;
```

Comment:
Indentation is off here (also, I've been trying to add explicit brackets in if-statements in new code)